### ISSUE TITLE
can't scroll columns

### ISSUE THREAD

**Issue by 0-wiz-0** [YS-L/csvlens#99]
I recently tried scrolling the columns in a big CSV but couldn't.
I found out that the problem was that the last line had fewer columns.
It surprised me at the time, so I thought I should mention it here.
Feel free to close it if you think this is too non-standard and doesn't have to be supported.

Here's an example file. In a 216 column terminal, whatever I do, I can't make csvlens display the last columns.
[foo.csv](https://github.com/user-attachments/files/16736275/foo.csv)

**Comment by YS-L** [comment]
Good catch! Support for irregular CSVs is on a best-effort basis, but this can be fixed without overly complex changes. I’ll push a fix shortly.

**Comment by 0-wiz-0** [comment]
Thank you!
